src/www/containers/userprofile/UserProfile.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from "react";
+import { withTranslation } from "react-i18next";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import "./UserProfile.scss";
+
+const minCardsShown = 3;
+
+// user.actionsAttended
+const actionsAttended = require("./actions.json");
+const actionsOrganized = require("./actions.json");
+const teams = require("./teams.json");
+
+const mapState = (state) => ({
+  user: state.auth.user,
+});
+
+export default connect(
+  mapState,
+  undefined
+)(
+  withTranslation()(
+    class UserProfile extends Component {
+      state = {
+        activeTab: "actions",
+        actionsAttended: {
+          list: actionsAttended,
+          shown: 3,
+        },
+        actionsOrganized: {
+          list: actionsOrganized,
+          shown: 3,
+        },
+        edcountitMode: "off",
```

Comment:
Something went wrong here :stuck_out_tongue: 
```suggestion
        editMode: false,
```